demos/python_demos/object_detection_demo_ssd_async/object_detection_demo_ssd_async.py

```diff
@@ -50,6 +50,53 @@ def build_argparser():
     return parser
 
 
+class SingleOutputPostprocessor:
+    def __init__(self, output_layer):
+        self.output_layer = output_layer
+
+    def __call__(self, outputs):
+        return outputs[self.output_layer][0][0]
+
+
+class MultipleOutputPostprocessor:
+    def __init__(self, bboxes_layer='bboxes', scores_layer='scores', labels_layer='labels'):
+        self.bboxes_layer = bboxes_layer
+        self.scores_layer = scores_layer
+        self.labels_layer = labels_layer
+
+    def __call__(self, outputs):
+        bboxes = outputs[self.bboxes_layer][0]
+        scores = outputs[self.scores_layer][0]
+        labels = outputs[self.labels_layer][0]
+        result = [[0, label, score, *bbox] for label, score, bbox in zip(labels, scores, bboxes)]
+        return result
```

Comment:
There's no reason to assign a value to a variable and immediately return that variable. Just return the value directly.